test/integration/attachment_access_limited_integration_test.rb

```diff
@@ -40,14 +40,67 @@ class AttachmentAccessLimitedIntegrationTest < ActionDispatch::IntegrationTest
 
         it "marks attachment as access limited in Asset Manager" do
           Services.asset_manager
-            .expects(:update_asset)
-            .at_least_once.with("asset-id", has_entry("access_limited", %w[user-uid]))
+                  .expects(:update_asset)
+                  .at_least_once.with("asset-id", has_entry("access_limited", %w[user-uid]))
 
           AssetManagerAttachmentMetadataWorker.drain
         end
       end
     end
 
+    #TODO work out where to move auth bypass id tests
+    context "given a consultation" do
+      let(:edition) { create(:draft_consultation) }
+
+      before do
+        setup_publishing_api_for(edition)
+
+        stub_whitehall_asset("logo.png", id: "asset-id", draft: true)
+
+        visit admin_consultation_path(edition)
+        click_link "Edit draft"
+        click_link "Final outcome"
+        fill_in "consultation_outcome[summary]", with: "Outcome of consultation"
+        click_button "Save"
+        click_link "Upload new file attachment"
+        fill_in "Title", with: "Outcome attachment"
+        attach_file "File", path_to_attachment("logo.png")
+        click_button "Save"
+        assert_text "Attachment 'Outcome attachment' uploaded"
+      end
+
+      it "sends the attachment to asset manager with the consultation's auth_bypass_id" do
+        Services.asset_manager.expects(:create_whitehall_asset).with(
+          has_entry(auth_bypass_ids: [edition.auth_bypass_id]),
+        )
+
+        AssetManagerCreateWhitehallAssetWorker.drain
+      end
+    end
+
+    context "given a case study" do
+      let(:edition) { create(:draft_case_study) }
+
+      before do
+        setup_publishing_api_for(edition)
+
+        stub_whitehall_asset("logo.png", id: "asset-id", draft: true)
+
+        visit admin_case_study_path(edition)
+        click_link "Edit draft"
+        choose "Use a custom image"
+        attach_file "File", path_to_attachment("minister-of-funk.960x640.jpg")
+        click_button "Save"
+      end
+
+      it "sends the attachment to asset manager with the case study's auth_bypass_id" do
```

Comment:
Would be good to clarify that this is a test for an image rather than an attachment.